tests/app/db.py

```diff
@@ -836,28 +836,28 @@ def ses_complaint_callback():
 def ses_notification_callback():
     return (
         '{\n  "Type" : "Notification",\n  "MessageId" : "ref1",'
-        '\n  "TopicArn" : "arn:aws:sns:eu-west-1:123456789012:testing",'
+        '\n  "TopicArn" : "arn:aws:sns:eu-west-2:123456789012:testing",'
         '\n  "Message" : "{\\"notificationType\\":\\"Delivery\\",'
         '\\"mail\\":{\\"timestamp\\":\\"2016-03-14T12:35:25.909Z\\",'
         '\\"source\\":\\"test@test-domain.com\\",'
-        '\\"sourceArn\\":\\"arn:aws:ses:eu-west-1:123456789012:identity/testing-notify\\",'
+        '\\"sourceArn\\":\\"arn:aws:ses:eu-west-2:123456789012:identity/testing-notify\\",'
         '\\"sendingAccountId\\":\\"123456789012\\",'
         '\\"messageId\\":\\"ref1\\",'
         '\\"destination\\":[\\"testing@digital.cabinet-office.gov.uk\\"]},'
         '\\"delivery\\":{\\"timestamp\\":\\"2016-03-14T12:35:26.567Z\\",'
         '\\"processingTimeMillis\\":658,'
         '\\"recipients\\":[\\"testing@digital.cabinet-office.gov.uk\\"],'
         '\\"smtpResponse\\":\\"250 2.0.0 OK 1457958926 uo5si26480932wjc.221 - gsmtp\\",'
-        '\\"reportingMTA\\":\\"a6-238.smtp-out.eu-west-1.amazonses.com\\"}}",'
+        '\\"reportingMTA\\":\\"a6-238.smtp-out.eu-west-2.amazonses.com\\"}}",'
```

Comment:
I think this is just testing fo it probably doesn't matter, but this isn't a valid domain, the eu-west-2 one appears to be a60-135.smtp-out.us-west-2.amazonses.com (There's a couple of other places you just replaces eu-west-1 with eu-west-2 in a domain which may also need changing)

Reply:
Endpoints and ARNs associated with SES will be corrected when we work on the SES integration ticket.